php.php

```diff
@@ -0,0 +1,18 @@
+<?php
+    class Deposit {
+
+        /**
+         * the deposit ownner.
+         *
+         * @var string witout info
+         */
+        private $_owner;
+
+        /*
+         * multiline coment
+         * goes ere
+         */
+        public function __construct() {
+            $a = 3; //inline coment
```

Comment:
Typo inside comment in word `coment`. Possible corrections:  `comment` (37), `moment` (30), `cement` (26)